### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Koyamot**
#This PR fixes Profile Select, Mission Control, and Child Dashboard styling

### CODE REVIEW

File: src/components/common/ChildCard.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function NewChildCard(props) {
   };
 
   return (
-    <div className="newChildCardContainer">
+    <div clanssName="newChildCardContainer">
```
Comment (depadiernos) [inline src/components/common/ChildCard.js]: Here's the typo I pointed out in Slack.

**Review by depadiernos — DISMISSED** [review]
lgtm

File: src/components/common/ChildCard.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ function NewChildCard(props) {
   };
 
   return (
-    <div className="newChildCardContainer">
+    <div classsName="newChildCardContainer">
```
Comment (depadiernos) [inline src/components/common/ChildCard.js]: 🤔 there's an extra 's' in classsName.

Comment (Koyamot) [reply src/components/common/ChildCard.js]: It just might not be meant to be Dennis!! 😭😂

Comment (Koyamot) [reply src/components/common/ChildCard.js]: Fixed it~!!!!

**Review by depadiernos — DISMISSED** [review]
Lgtm

**Review by Robert-Lark — APPROVED** [review]
Looks great

**Pull request merged**